coursier.bzl

```diff
@@ -862,7 +905,9 @@ def _coursier_fetch_impl(repository_ctx):
         artifact.update({"sha256": shas[str(repository_ctx.path(file))]})
 
     dep_tree.update({
-        "__AUTOGENERATED_FILE_DO_NOT_MODIFY_THIS_FILE_MANUALLY": _compute_dependency_tree_signature(dep_tree["dependencies"]),
+        "__AUTOGENERATED_FILE_DO_NOT_MODIFY_THIS_FILE_MANUALLY": "THERE_IS_NO_DATA_ONLY_ZUUL",
+        "__RESOLVED_ARTIFACTS_HASH": _compute_dependency_tree_signature(dep_tree["dependencies"]),
```

Comment:
Should the`__RESOLVED_ARTIFACTS_HASH` and `__INPUT_ARTIFACTS_HASH ` entries only be added if `fail_if_repin_required` is set to true?

Reply:
I've retained the current behaviour we already have. I'd prefer not to change that in this PR, as we're already doing enough stuff here.